lib/src/base-styles/_nx-lists.scss

```diff
@@ -128,109 +126,77 @@
-  }
-
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
-      pointer-events: none;
+      &, &:hover {
+        background-color: #d3e6ff;
+      }
     }
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
-      &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
-        cursor: pointer;
-        outline-style: none; // removes :focus outline
-
-        .fa-chevron-right {
-          color: $nx-color-blue-41;
-        }
-      }
+    &:hover {
+      background-color: #f1f3fa;
+      cursor: pointer;
+      outline-style: none; // removes :focus outline
+    }
 
-      &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-      }
+    &:active {
+      background-color: #d3e6ff;
+    }
+
+    &:focus {
+      box-shadow: 0 0 3px 1px rgba(2, 166, 217, 0.3);
+      border: solid 1px #02a6d9;
+      outline: none;
     }
 
     &.nx-list__item--empty {
       @include regular();
 
-      border-top: $nx-border; // assume it's always the first item
       color: $nx-text-color-light;
     }
 
-    &.nx-error {
-      color: $nx-text-color-error;
-
-      &:not(.nx-disabled):not([disabled]) {
-        border: $nx-border-error;
+    &.disabled {
+      color: $nx-text-color-disabled;
 
-        &:hover, &:focus, &:active {
-          background-color: $nx-fail-color-bg;
-          border: $nx-border-error;
-          color: $nx-text-color-error;
-          cursor: default;
-        }
+      &:hover {
+        background-color: transparent;
+        cursor: default;
       }
-    }
 
-    .nx-list__subtext {
-      pointer-events: none; // Fix for IE11 :active state display
+      > .nx-chevron {
+        display: none;
+      }
     }
 
-    &.nx-list__item--disabled {
-      color: $nx-text-color-disabled;
-      cursor: default;
+    > .nx-chevron {
+      align-self: center;
+      color: #2c2c2c;
+      -ms-grid-row: 1;
+      -ms-grid-column: 2;
+      grid-area: actions;
 
-      &:hover {
-        border-top: 1px solid transparent;
-        border-bottom: $nx-border-row;
-      }
+      // custom sizing for the icon to match the text size
+      height: 25px;
+      width: 1em;
     }
   }
-
-  .fa-chevron-right {
-    margin-right: 3px;
-    margin-top: 1px;
-  }
 }
 
-/*
- * Definition Lists
- *
- * Typically use <dl>
- */
-
-.nx-list--definition-list {
-  box-sizing: border-box;
-
-  display: flex;
-  flex-wrap: wrap;
-
-  .nx-list__item {
-    @include regular();
+.nx-list__actions {
+  @include container-horizontal;
 
-    display: inline-block;
-    width: 75%;
+  -ms-grid-row: 1;
+  -ms-grid-column: 2;
+  grid-area: actions;
+  -ms-grid-column-align: end;
```

Comment:
This doesn't seem necessary either.

Reply:
> This doesn't seem necessary either.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/e57ab0da139398c2561b16f90a4e9976f7da0c73